api/handlers.go

```diff
@@ -668,19 +911,9 @@ func SearchStatsHandler(c *gin.Context) {
 	if err != nil {
 		return
 	}
-	skipUids, _, err := GetNotInTagsIds(cacheM.TagsStats().GetTree(), db)
```

Comment:
@bbfsdev You don't need it?

Reply:
It was accidental. Thank you for pointing out. Reverted back.